metabase/metabase
# Fix histogram x axis ticks

Closes https://github.com/metabase/metabase/issues/41148

### Description

Fixes histogram ticks when rotation to 45 or 90 degrees is enabled. Auto-rotation also works now.

After the rebase on top of the master storybook stories font has changed but it is not related to this PR..

### How to verify

- New -> Orders by Total: 10 buckets
- Ensure ticks are horizontal
- Change to 50 buckets
- Ensure ticks are rotated to 45 degrees
- Change to 100 buckets
- Ensure ticks are rotated to 90 degrees

### Demo

https://github.com/metabase/metabase/assets/14301985/3e215101-f056-466c-949e-2a88b759d67c

### Checklist
